### PR TITLE
2.0 view and schema

### PR DESCRIPTION
**Pull request by BillyAbildgaard**
@RackHD/corecommitters @dalebremner @keedya @geoff-reid @srinia6

### CODE REVIEW

File: static/monorail-2.0.yaml [hunk 1]
```diff
@@ -2328,6 +2328,7 @@ paths:
       operationId: nodesGetTagsById
       x-privileges: [ 'Read' ]
       x-authentication-type: [ 'jwt' ]
+ #     x-view: tags.2.0.json
```
Comment (anhou) [inline static/monorail-2.0.yaml]: if don't need this line, remove it

File: static/schemas/2.0/tags.2.0.json [hunk 2]
```diff
@@ -0,0 +1,23 @@
+{
+    "id": "tags.2.0",
+    "title": "Tags.2.0",
+    "definitions": {
+        "TagsPost": {
+	    "additionalProperties": false,
```
Comment (anhou) [inline static/schemas/2.0/tags.2.0.json]: indent

**Comment by anhou** [comment]
:+1:

**Comment by yyscamper** [comment]
:+1: after fix @anhou's comment.

**Comment by srinia6** [comment]
+1

**Comment by BillyAbildgaard** [comment]
test this please

**Comment by BillyAbildgaard** [comment]
test this please

**Comment by BillyAbildgaard** [comment]
jenkins...test this please

**Pull request merged**